src/CronExpression.php

```diff
@@ -18,13 +18,52 @@ class CronExpression
      */
     protected $testTime;
 
-    public function shouldRun(string $expression): bool
+    /**
+     * Allows us to set global timezone for all tasks
+     * on construct
+     * 
+     * @param string $timezone
+     * 
+     * @return void
+     */
+    public function __construct(string $timezone = null)
+    {
+        if($timezone === null){
```

Comment:
The goal is for this to make it into `codeigniter4/codeigniter4` eventually so should follow the same style guide:
```
if ($timezone === null)
{
```

Reply:
Same thing throughout these changes.